content/docs/Tutorials/basic/_index.md

```diff
@@ -57,7 +57,7 @@ While running SUSHI, there will be a series of informational messages. When SUSH
 ║ ╰──────────┴────────────┴───────────┴─────────────┴───────────╯ ║
 ║                                                                 ║
 ╠═════════════════════════════════════════════════════════════════╣
-║ Nice! You're totally krilling it!      0 Errors      0 Warnings ║
+║ Nice! You're totally killing it!      0 Errors      0 Warnings ║
```

Comment:
```suggestion
║ You are dolphinitely doing great!      0 Errors      0 Warnings ║
```
Since we always use a pun, I'd like to swap out the inappropriate one with this one instead.  (Sorry -- we did not realize "krilling" had a slang meaning!).